# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/import/cpan/cpan-device-serialport.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN Device-SerialPort-1.002/Makefile.PL Device-SerialPort-1.002-new/Makefile.PL
--- Device-SerialPort-1.002/Makefile.PL	2004-04-06 18:14:42.000000000 +0200
+++ Device-SerialPort-1.002-new/Makefile.PL	2005-01-04 10:49:38.003585616 +0100
@@ -83,9 +83,11 @@
 }
 
 if (!defined($file)) {
-    die "Could not find a serial port to use for testing.\n".
+    warn "Could not find a serial port to use for testing.\n".
         "Please specify one on the 'perl Makefile.PL' command line, like so:\n".
-	"\tperl Makefile.PL /dev/ttyS0\n";
+	"\tperl Makefile.PL /dev/ttyS0\n".
+	"ROCKLinux will now set the file to test to /dev/null\n";
+	$file = "/dev/null";
 }
 
 my $dfile = "t/DefaultPort.pm";
